.github/templates/windows_ci_workflow.yml.j2

```diff
@@ -83,12 +83,38 @@ jobs:
           name: ${{ env.BUILD_ENVIRONMENT }}
           path: C:\w\build-results
 
+  generate-test-matrix:
+    if: ${{ github.repository_owner == 'pytorch' }}
+    runs-on: ubuntu-18.04
+    env:
+      NUM_TEST_SHARDS: !{{ num_test_shards }}
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    container:
+      image: python:3.9
+    steps:
+      - name: Clone pytorch/pytorch
+        uses: actions/checkout@v2
+      - name: Generating test matrix
+        id: set-matrix
+        run: |
+          # outputting for debugging purposes
+          MATRIX=$(python .github/scripts/generate_pytorch_test_matrix.py)
+          echo "${MATRIX}"
+          echo "::set-output name=matrix::${MATRIX}"
```

Comment:
is this change meant to be included in this PR, since the title says it's only for Linux?

Reply:
Yes, I figured it should get reviewed once as it's the same code, and I can just enable windows in a separate PR.